src/components/Assay/AssayLineGraph.tsx

```diff
@@ -136,8 +136,9 @@ class AssayLineGraph extends React.Component<AssayLineProps, AssayLineState> {
       if (i === lockedAssays.length - 1) {
         height += 30;
       }
-      const yAxisFontSize = lockedAssays.length < 3 ? 12 :
-      lockedAssays.length < 4 ? 10 : 8;
+      const yAxisFontSize = lockedAssays.length < 3 
```

Comment:
I know I've not been consistent about styling ternary operators...but I think this looks a little nicer

Reply:
Yup, you're right.